src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,99 @@
+package lotto.controller;
+
+import lotto.domain.*;
+import lotto.domain.ticketfactory.FixedNumberTicketFactory;
+import lotto.exception.LottoCustomException;
+import lotto.utils.ValidateUtils;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoController {
+    private static final int EXIST = 1;
+    private static final String SEPARATOR = ",";
+    private final InputView inputView;
+
+    public LottoController(final Scanner scanner) {
+        inputView = new InputView(scanner);
+    }
+
+    public void run() {
+        Money money = inputMoney();
+        LottoTickets lottoTickets = buyLottoTickets(money);
+
+        LottoTicket winningTicket = makeWinningTicket();
+        LottoNumber bonusBall = makeBonusNumber(winningTicket);
+
+        showResult(money, lottoTickets, winningTicket, bonusBall);
+        ifChangeExist(money);
+    }
+
+    private void ifChangeExist(Money money) {
+        if (money.getChange() >= EXIST) {
+            OutputView.printGiveChange(money.getChange());
+        }
+    }
+
+    private Money inputMoney() {
+        try {
+            OutputView.printMoneyMessage();
+            Money money = new Money(ValidateUtils.parseInt(inputView.inputValue()));
+            OutputView.printTicketCountMessage(money.getTicketCount());
+            return money;
+        } catch (LottoCustomException e) {
+            OutputView.printErrorMessage(e.getMessage());
+            return inputMoney();
+        }
+    }
+
+    private LottoTickets buyLottoTickets(Money money) {
+        LottoTickets lottoTickets = new LottoTickets();
+        lottoTickets.makeTicketByCount(money.getTicketCount());
+        OutputView.printAllTickets(lottoTickets);
+        return lottoTickets;
+    }
+
+    private LottoTicket makeWinningTicket() {
+        try {
+            OutputView.printWinningNumbers();
+            return FixedNumberTicketFactory
+                    .makeTicket(splitWinningNumbers(inputView.inputValue()));
+        } catch (LottoCustomException e) {
+            OutputView.printErrorMessage(e.getMessage());
+            return makeWinningTicket();
+        }
+    }
+
+    private Set<String> splitWinningNumbers(String winningNumbers) {
+        return new HashSet<>(
+                Arrays.asList(winningNumbers.split(SEPARATOR))
+                .stream()
+                .map(String::trim)
+                .collect(Collectors.toList())
+        );
```

Comment:
```suggestion
        return Arrays.stream(winningNumbers.split(SEPARATOR))
            .map(String::trim)
            .collect(Collectors.toSet());
```

1. 인텔리제이가 멋진 코드를 제안해 주네요 😅 기존 코드를 보면 배열을 리스트로 바꾸고, 스트림 연산을 한 후 다시 리스트로 묶었다가 셋으로 바꾸고 있어요. 물론 앞으로 점차 익숙해지겠지만, 스트림과 컬렉션에 관해 조금 더 공부해 보시면 좋을 것 같아요~
2. 이 정도는 `InputView`에서 처리해 줘도 되지 않을까요? 컨트롤러의 책임은 어디까지일까요?

Reply:
여전히 제가 mvc를 잘 모르는 것이 이번 로또 미션까지 넘어오는 것 같습니다 ••• 으악!! ㅋㅋ 컨트롤러는 뷰와 모델을 이어주는 역할을 하는 아이인데••• 입력값을 원하는 자료형으로 변환하는 과정은 inputView에서 자체적으로 처리해줘도 관계없나요? 사실 원래는 Inputview에서 처리해서 Set<>으로 리턴하도록 작성되었었는데, 무언가 inputWinningNumbers()를 호출하면 왠지 입력만 받아야할 것 같고, 결과값으로 Set을 돌려주는게 적절치 않다고 스스로 생각해서 controller로 메서드를 빼왔거든요.